install.sh

```diff
@@ -21,12 +21,17 @@ else
 fi
 
 # Stay backwards compatible with older versions that had the version in the file name.
-if [[ "v1v2v3" == *"$VERSION"* ]]; then
+if [[ "v1v2v3v4" == *"$VERSION"* ]]; then
```

Comment:
As soon as we have all pointing to 'latest' we need to remove this and make latest the new v1, but I will make the honors next week :)